ieeeuoft/makeuoft-site-2024
# Merge QR code scanning changes from MakeUofT 2023

## Overview

- merge scanner changes from makeuoft 2023

## Unit Tests Created

- n/a


## Steps to QA

- test qr code scanning
  - run migrations
  - sign in as an admin user on localhost:8000, you should see a different view
  - if styling looks weird make sure to collect static and run scss watcher(refer to README for specifics)
 - make sure HSS or frontend works properly, give yourself HSS Test User Permission on Django to bypass restrictions(for ex, hss not open yet and etc)
